apps/tangle-dapp/components/Lists/AssetList.tsx

```diff
@@ -120,7 +122,7 @@ export const AssetList = ({
                 className="cursor-default text-mono-200 dark:text-mono-0"
               >
                 {asset.balance
-                  ? `${getRoundedAmountString(asset.balance.toNumber(), 4)} ${asset.symbol}`
+                  ? `${asset.balance} ${asset.symbol}`
```

Comment:
Should we be formatting the balance here for readability? In case it's low or high balances, it may show up as `1000000`